app/views/categories/show.html.erb

```diff
@@ -0,0 +1,96 @@
+
+<body class="animsition">
+  <section class="bg-title-page p-t-50 p-b-40 flex-col-c-m category_wall">
+    <h2 class="l-text2 t-center">
+      Category
+    </h2>
+    <p class="m-text13 t-center">
+      New Comics Collection 2018
```

Comment:
đưa vào I18n hoặc config

Reply:
Phần này em xin phép được làm trong nhánh sau ạ. Vì cái này em đang demo in ra của phần truyện, chưa hoàn thành ạ